source/postprocess/tracers.cc

```diff
@@ -0,0 +1,406 @@
+
+      if (SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(generator.get()))
+        sim->initialize_simulator (this->get_simulator());
+      generator->parse_parameters(prm);
+
+
+      // Create an output object depending on what the parameters specify
+      output.reset(Particle::Output::create_particle_output<dim>
+                   (prm));
+
+      // We allow to not generate any output plugin, in which case output is
+      // a null pointer. Only initialize output if it was created.
+      if (output)
+        {
+          if (SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(output.get()))
+            sim->initialize_simulator (this->get_simulator());
+          output->initialize();
+        }
+
+
+      // Create an integrator object depending on the specified parameter
+      std_cxx11::shared_ptr<Particle::Integrator::Interface<dim> >   integrator
+      (Particle::Integrator::create_particle_integrator<dim> (prm));
+
+      if (SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(integrator.get()))
+        sim->initialize_simulator (this->get_simulator());
+      integrator->parse_parameters(prm);
+
+      // Create an interpolator object depending on the specified parameter
+      std_cxx11::shared_ptr<Particle::Interpolator::Interface<dim> > interpolator
+      (Particle::Interpolator::create_particle_interpolator<dim> (prm));
+      if (SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(interpolator.get()))
+        sim->initialize_simulator (this->get_simulator());
+      interpolator->parse_parameters(prm);
+
+
+      // Creaty an property_manager object and initialize its properties
+      std_cxx11::shared_ptr<Particle::Property::Manager<dim> > property_manager (new Particle::Property::Manager<dim> ());
+      SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(property_manager.get());
+      sim->initialize_simulator (this->get_simulator());
+      property_manager->parse_parameters(prm);
+      property_manager->initialize();
+
+
+      // Initialize the particle world with the appropriate settings
+      // Ownership of generator, integrator, interpolator and property_manager
+      // is transferred to world here
+      if (SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(&world))
+        sim->initialize_simulator (this->get_simulator());
+      world.initialize(generator,
+                       integrator,
+                       interpolator,
+                       property_manager,
+                       load_balancing,
+                       max_tracers_per_cell,
+                       tracer_weight);
+    }
+  }
+}
+
+
+// explicit instantiations
+namespace aspect
+{
+  namespace Postprocess
+  {
+    ASPECT_REGISTER_POSTPROCESSOR(Tracers,
+                                  "tracers",
+                                  "A Postprocessor that creates tracer particles, which follow the "
```

Comment:
"tracer particles, which" -> "tracer particles that"

(I used to write this all the time, but it's a Germanism. In particular the use of the comma.)